modules/books.js

```diff
@@ -0,0 +1,28 @@
+/* eslint-disable import/no-cycle */
+/* eslint-disable no-restricted-globals */
```

Comment:
- [ ] As the previous code reviewer mentioned, please fix the linters and remove the `eslint-disable` lines from the Javascript files.